packages/components/src/List/ListComposition/Manager/ListManager.component.test.js

```diff
@@ -91,10 +96,42 @@ describe('List Manager', () => {
 		expect(wrapper.find(TestConsumer).prop('collection')).toEqual([{ id: 0, name: 'toto' }]);
 	});
 
+	it('should propagate filtered column list', () => {
+		// given
+		const wrapper = mount(
+			<ListManager
+				collection={[
+					{ dataKey: 'id', label: 'ID' },
+					{ dataKey: 'name', label: 'Name' },
+				]}
+			>
+				<ContextTestConsumer />
+			</ListManager>,
+		);
+		expect(wrapper.find(TestConsumer).prop('filteredColumns')).toBeUndefined();
+
+		const filteredColumns = ['name'];
+
+		// when
+		act(() => {
+			const setFilteredColumns = wrapper.find(TestConsumer).prop('setFilteredColumns');
+			setFilteredColumns(filteredColumns);
+		});
+		wrapper.update();
+
+		// then
+		expect(wrapper.find(TestConsumer).prop('filteredColumns')).toBe(filteredColumns);
+	});
+
```

Comment:
Please don't do this kind of tests, that test an internal implementation detail.